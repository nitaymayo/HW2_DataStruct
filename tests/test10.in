add_horse 574013 4
follow 574013 574013
add_horse 764882 7
add_horse 985261 25
add_horse 422243 28
add_horse 117900 1
get_speed 117900
leads 117900 764882
add_horse 103446 14
get_speed 52759
get_speed 985261
add_horse 929684 8
get_speed 764882
add_horse 381604 2
add_horse 269138 13
add_horse 372751 14
add_herd 225833
get_speed 381604
join_herd 117900 225833
add_horse 455883 7
can_run_together 225833
add_horse 913990 9
can_run_together 225833
add_horse 674368 8
add_horse 558122 11
can_run_together 225833
add_horse 379649 11
can_run_together 225833
add_horse 125994 1
get_speed 381604
leads 381604 103446
can_run_together 225833
join_herd 372751 225833
add_horse 663986 17
join_herd 103446 225833
get_speed 809729
join_herd 455883 225833
can_run_together 225833
add_horse 833599 25
can_run_together 225833
join_herd 125994 225833
join_herd 574013 225833
add_horse 234653 11
join_herd 764882 225833
can_run_together 225833
leads 674368 929684
can_run_together 225833
can_run_together 225833
join_herd 663986 225833
add_horse 856719 24
leads 558122 764882
can_run_together 225833
leads 764882 103446
leads 663986 455883
leads 574013 663986
leads 125994 455883
follow 117900 103446
follow 372751 372751
follow 103446 372751
follow 455883 455883
follow 125994 117900
follow 574013 574013
follow 764882 764882
follow 663986 103446
can_run_together 225833
leads 103446 125994
leads 574013 103446
leads 103446 125994
leads 125994 663986
leads 574013 125994
join_herd 422243 225833
get_speed 558122
get_speed 856719
get_speed 929684
leads 455883 929684
add_horse 261350 16
add_horse 717931 10
add_horse 742276 9
join_herd 261350 225833
get_speed 234653
follow 985261 764882
get_speed 764882
join_herd 234653 225833
add_horse 473322 1
join_herd 473322 225833
get_speed 473322
leads 234653 913990
add_horse 201882 9
leave_herd 809729
can_run_together 225833
get_speed 764882
add_horse 473145 26
join_herd 674368 225833
leads 473145 717931
leads 913990 742276
leads 473145 201882
get_speed 473145
add_horse 827396 12
add_horse 541129 8
add_horse 512757 5
add_horse 533827 6
add_horse 249001 6
add_horse 654672 24
join_herd 833599 225833
add_horse 548730 20
join_herd 541129 225833
join_herd 473145 225833
get_speed 455883
leave_herd 261350
leads 533827 379649
leave_herd 473322
leave_herd 541129
join_herd 512757 225833
can_run_together 225833
leads 103446 663986
leads 833599 512757
leads 455883 234653
leads 234653 833599
leads 125994 473145
leads 455883 663986
leads 125994 674368
follow 764882 372751
follow 372751 674368
follow 674368 234653
follow 234653 103446
follow 422243 125994
follow 663986 512757
follow 512757 473145
follow 473145 833599
follow 833599 117900
follow 574013 455883
follow 103446 512757
follow 455883 512757
follow 125994 833599
follow 117900 455883
can_run_together 225833
leads 663986 125994
leads 372751 234653
leads 117900 455883
leads 372751 833599
leads 764882 117900
leads 574013 422243
leads 574013 422243
join_herd 929684 225833
add_horse 361099 15
add_horse 988606 13
get_speed 249001
leads 125994 827396
add_horse 747161 27
add_horse 377910 29
join_herd 717931 225833
join_herd 249001 225833
join_herd 381604 225833
leads 249001 473145
add_horse 235759 23
join_herd 717931 225833
add_herd 371275
get_speed 201882
add_horse 7998 29
add_horse 251661 11
add_horse 612673 15
leads 7998 7998
get_speed 663986
add_horse 327359 24
get_speed 747161
can_run_together 225833
leads 125994 663986
leads 717931 422243
leads 117900 249001
leads 574013 512757
leads 473145 663986
leads 674368 663986
leads 764882 372751
leads 234653 674368
leads 372751 249001
leads 674368 249001
follow 103446 422243
leave_herd 422243
join_herd 422243 225833
follow 929684 717931
follow 234653 929684
follow 455883 234653
follow 764882 717931
follow 833599 455883
follow 249001 473145
follow 125994 473145
follow 574013 103446
follow 372751 717931
follow 381604 717931
follow 663986 249001
follow 422243 473145
follow 117900 512757
follow 512757 833599
follow 717931 473145
follow 674368 125994
follow 473145 674368
can_run_together 225833
leads 663986 117900
leads 125994 249001
leads 717931 422243
leads 381604 833599
leads 234653 249001
leads 125994 381604
leads 473145 117900
leads 117900 455883
leads 512757 674368
leads 117900 674368
add_horse 749239 24
add_herd 215872
can_run_together 225833
add_horse 169275 29
add_horse 122976 3
add_horse 763885 21
leads 327359 913990
join_herd 742276 215872
add_horse 553367 29
get_speed 422243
leave_herd 654672
add_horse 1046128 28
join_herd 742276 215872
join_herd 377910 371275
get_speed 122976
can_run_together 225833
leads 512757 422243
leads 717931 663986
leads 103446 117900
leads 473145 234653
leads 117900 455883
leads 663986 125994
leads 674368 512757
leads 833599 473145
leads 674368 717931
follow 764882 717931
leave_herd 717931
join_herd 717931 225833
follow 117900 249001
follow 372751 455883
follow 103446 512757
follow 455883 234653
follow 125994 574013
follow 574013 125994
follow 663986 117900
follow 234653 929684
follow 674368 833599
follow 833599 117900
follow 473145 512757
follow 512757 422243
follow 929684 473145
follow 717931 663986
follow 249001 512757
follow 381604 663986
follow 422243 234653
can_run_together 225833
leads 574013 117900
leads 372751 125994
leads 125994 717931
leads 663986 512757
leads 103446 574013
leads 764882 117900
leads 574013 125994
leads 473145 125994
leads 764882 381604
get_speed 988606
join_herd 7998 371275
add_horse 818455 30
add_horse 842868 28
can_run_together 225833
leads 381604 663986
leads 372751 929684
leads 125994 764882
leads 234653 574013
leads 929684 372751
leads 663986 455883
leads 234653 674368
leads 455883 381604
leads 125994 381604
leads 381604 249001
leads 125994 455883
follow 455883 674368
follow 674368 234653
follow 234653 929684
follow 929684 381604
follow 381604 422243
follow 422243 372751
follow 372751 455883
follow 764882 717931
follow 717931 663986
follow 663986 833599
follow 833599 473145
follow 473145 249001
follow 249001 574013
follow 574013 125994
follow 125994 103446
follow 103446 117900
follow 117900 512757
follow 512757 764882
can_run_together 225833
leads 764882 663986
leads 372751 473145
leads 117900 764882
leads 381604 833599
leads 103446 473145
leads 372751 574013
leads 674368 833599
leads 833599 249001
leads 473145 717931
can_run_together 371275
add_horse 679566 19
join_herd 913990 225833
can_run_together 215872
add_horse 934671 16
get_speed 379649
add_horse 803181 18
add_horse 590845 24
add_horse 782803 5
join_herd 533827 225833
add_horse 307075 13
join_herd 749239 225833
join_herd 654672 371275
get_speed 913990
add_horse 56486 10
add_horse 988045 26
get_speed 842868
add_horse 572984 18
add_horse 799575 24