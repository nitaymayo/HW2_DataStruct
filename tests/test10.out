add_horse: SUCCESS
follow: INVALID_INPUT
add_horse: SUCCESS
add_horse: SUCCESS
add_horse: SUCCESS
add_horse: SUCCESS
get_speed: SUCCESS, 1
leads: SUCCESS, False
add_horse: SUCCESS
get_speed: FAILURE
get_speed: SUCCESS, 25
add_horse: SUCCESS
get_speed: SUCCESS, 7
add_horse: SUCCESS
add_horse: SUCCESS
add_horse: SUCCESS
add_herd: SUCCESS
get_speed: SUCCESS, 2
join_herd: SUCCESS
add_horse: SUCCESS
can_run_together: SUCCESS, True
add_horse: SUCCESS
can_run_together: SUCCESS, True
add_horse: SUCCESS
add_horse: SUCCESS
can_run_together: SUCCESS, True
add_horse: SUCCESS
can_run_together: SUCCESS, True
add_horse: SUCCESS
get_speed: SUCCESS, 2
leads: SUCCESS, False
can_run_together: SUCCESS, True
join_herd: SUCCESS
add_horse: SUCCESS
join_herd: SUCCESS
get_speed: FAILURE
join_herd: SUCCESS
can_run_together: SUCCESS, False
add_horse: SUCCESS
can_run_together: SUCCESS, False
join_herd: SUCCESS
join_herd: SUCCESS
add_horse: SUCCESS
join_herd: SUCCESS
can_run_together: SUCCESS, False
leads: SUCCESS, False
can_run_together: SUCCESS, False
can_run_together: SUCCESS, False
join_herd: SUCCESS
add_horse: SUCCESS
leads: SUCCESS, False
can_run_together: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
follow: SUCCESS
follow: INVALID_INPUT
follow: SUCCESS
follow: INVALID_INPUT
follow: SUCCESS
follow: INVALID_INPUT
follow: INVALID_INPUT
follow: SUCCESS
can_run_together: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
join_herd: SUCCESS
get_speed: SUCCESS, 11
get_speed: SUCCESS, 24
get_speed: SUCCESS, 8
leads: SUCCESS, False
add_horse: SUCCESS
add_horse: SUCCESS
add_horse: SUCCESS
join_herd: SUCCESS
get_speed: SUCCESS, 11
follow: FAILURE
get_speed: SUCCESS, 7
join_herd: SUCCESS
add_horse: SUCCESS
join_herd: SUCCESS
get_speed: SUCCESS, 1
leads: SUCCESS, False
add_horse: SUCCESS
leave_herd: FAILURE
can_run_together: SUCCESS, False
get_speed: SUCCESS, 7
add_horse: SUCCESS
join_herd: SUCCESS
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
get_speed: SUCCESS, 26
add_horse: SUCCESS
add_horse: SUCCESS
add_horse: SUCCESS
add_horse: SUCCESS
add_horse: SUCCESS
add_horse: SUCCESS
join_herd: SUCCESS
add_horse: SUCCESS
join_herd: SUCCESS
join_herd: SUCCESS
get_speed: SUCCESS, 7
leave_herd: SUCCESS
leads: SUCCESS, False
leave_herd: SUCCESS
leave_herd: SUCCESS
join_herd: SUCCESS
can_run_together: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
can_run_together: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, True
leads: SUCCESS, True
leads: SUCCESS, True
leads: SUCCESS, True
leads: SUCCESS, False
leads: SUCCESS, False
join_herd: SUCCESS
add_horse: SUCCESS
add_horse: SUCCESS
get_speed: SUCCESS, 6
leads: SUCCESS, False
add_horse: SUCCESS
add_horse: SUCCESS
join_herd: SUCCESS
join_herd: SUCCESS
join_herd: SUCCESS
leads: SUCCESS, False
add_horse: SUCCESS
join_herd: FAILURE
add_herd: SUCCESS
get_speed: SUCCESS, 9
add_horse: SUCCESS
add_horse: SUCCESS
add_horse: SUCCESS
leads: INVALID_INPUT
get_speed: SUCCESS, 17
add_horse: SUCCESS
get_speed: SUCCESS, 27
can_run_together: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, True
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, True
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
follow: SUCCESS
leave_herd: SUCCESS
join_herd: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
can_run_together: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, True
leads: SUCCESS, True
leads: SUCCESS, True
add_horse: SUCCESS
add_herd: SUCCESS
can_run_together: SUCCESS, False
add_horse: SUCCESS
add_horse: SUCCESS
add_horse: SUCCESS
leads: SUCCESS, False
join_herd: SUCCESS
add_horse: SUCCESS
get_speed: SUCCESS, 28
leave_herd: FAILURE
add_horse: SUCCESS
join_herd: FAILURE
join_herd: SUCCESS
get_speed: SUCCESS, 3
can_run_together: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, True
leads: SUCCESS, True
leads: SUCCESS, False
leads: SUCCESS, True
leads: SUCCESS, False
follow: SUCCESS
leave_herd: SUCCESS
join_herd: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
can_run_together: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, True
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, True
leads: SUCCESS, False
leads: SUCCESS, False
get_speed: SUCCESS, 13
join_herd: SUCCESS
add_horse: SUCCESS
add_horse: SUCCESS
can_run_together: SUCCESS, False
leads: SUCCESS, True
leads: SUCCESS, True
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, True
leads: SUCCESS, False
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
follow: SUCCESS
can_run_together: SUCCESS, False
leads: SUCCESS, True
leads: SUCCESS, False
leads: SUCCESS, True
leads: SUCCESS, False
leads: SUCCESS, True
leads: SUCCESS, False
leads: SUCCESS, False
leads: SUCCESS, True
leads: SUCCESS, True
can_run_together: SUCCESS, False
add_horse: SUCCESS
join_herd: SUCCESS
can_run_together: SUCCESS, True
add_horse: SUCCESS
get_speed: SUCCESS, 11
add_horse: SUCCESS
add_horse: SUCCESS
add_horse: SUCCESS
join_herd: SUCCESS
add_horse: SUCCESS
join_herd: SUCCESS
join_herd: SUCCESS
get_speed: SUCCESS, 9
add_horse: SUCCESS
add_horse: SUCCESS
get_speed: SUCCESS, 28
add_horse: SUCCESS
add_horse: SUCCESS
